viroconcom/read_write.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import csv
+
+
+def read_dataset(path='datasets/1year_dataset_A.txt'):

```

Comment:
This function is intended to "to load data in a format specified by viroconcom " as you are describing it. However, I agree that this is not clear in the code or documentation yet.

Could we proceed with this for the plot.py merge and open an issue to address the input / output format?

Reply:
What i mean is, that viroconcom should not enforce a file format. Rather it should use numpy/pandas/python objects as in- and output and leave the loading and writing of data to the user.

A compromise could be to rename the function:
- if it should load just one reference dataset it could be called load_<name of the dataset>
- if i should load files in a special format it could be called load_<name of file format>

Reply:
I think viroconcom should also specify a standard format for input/output. Of course we could also make it easy to import other file formats, but it makes things easier if a standard format is defined.

Let's discuss this as part of a seperate issue / featur eplease.